app/views/updates/update_files/_nqt_job_alerts_2020_07_29.html.erb

```diff
@@ -1,2 +1,3 @@
-Newly qualified teachers (NQTs) can now set up a [job alert](https://teaching-vacancies.service.gov.uk/sign-up-for-NQT-job-alerts) that will notify them of any jobs that are suitable for NQTs. 
+Newly qualified teachers (NQTs) can now set up a <%= link_to “job alerts”, "https://teaching-vacancies.service.gov.uk/sign-up-for-NQT-job-alerts"
```

Comment:
Nice!! `<%= link_to "job alert", nqt_job_alerts_path %>` means we don't need to hardcode the link

Reply:
Whoo-hoo!! Aw, I'm pleased that wasn't a disaster.

Reply:
Connor has correctly got rid of the 's' in 'job alerts'.

Reply:
(Connor is using the straight ones)

Reply:
I predict that is why the review app's update page is kaputt: https://teaching-vacancies-review-pr-2028.london.cloudapps.digital/updates

Reply:
Doh. I cut and pasted - I shoulda better than known that. Still, overall, it was not too shabby for a content designer!

Reply:
I don't know if the closing tag `%>` being on the following line will cause issues, but it's neater to have it on the same line.